use_timeout: true
baudrate: 460800  # 9600 19200 38400 57600 115200 230400 460800 500000 576000 921600
